Mathlib/Data/Sym/Basic.lean

```diff
@@ -532,6 +532,14 @@ theorem mem_append_iff {s' : Sym α m} : a ∈ s.append s' ↔ a ∈ s ∨ a ∈
   Multiset.mem_add
 #align sym.mem_append_iff Sym.mem_append_iff
 
+open Multiset Classical in
+@[simps apply]
+noncomputable def one_equiv : α ≃ Sym α 1 where
```

Comment:
According to naming convention this should be named `oneEquiv`. But I'd expect `oneEquiv` to refer to `Sym α 1 ≃ α` instead. It's also possible to make this computable (to follow)
```suggestion
noncomputable def one_equiv : α ≃ Sym α 1 where
```

Reply:
Here's a computable definition:
```
noncomputable def oneEquiv : α ≃ Sym α 1 where
  toFun a := ⟨{a}, by simp⟩
  invFun s := Quotient.liftOn (Equiv.subtypeQuotientEquivQuotientSubtype 
      (·.length = 1) _ (fun l ↦ Iff.rfl) (fun l l' ↦ by rfl) s)
    (fun l ↦ l.1.head <| List.length_pos.mp <| by linarith [l.2])
    fun ⟨l, h⟩ ⟨l', h'⟩ _perm ↦ by
      obtain ⟨a, rfl⟩ := List.length_eq_one.mp h'
      exact List.eq_of_mem_singleton (_perm.mem_iff.mp <| List.head_mem _)
  left_inv a := by rfl
  right_inv := by rintro ⟨⟨l⟩, h⟩; ext; obtain ⟨a, rfl⟩ := List.length_eq_one.mp h; rfl
```

Reply:
Ooh thank you, this is great!